setup.py

```diff
@@ -1,21 +1,14 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
-""" niworkflows setup script """
+"""niworkflows setup script."""
 import sys
 from setuptools import setup
 import versioneer
 
-# Give setuptools a hint to complain if it's too old a version
-# 30.3.0 allows us to put most metadata in setup.cfg
-# 30.4.0 gives us options.packages.find
-# Should match pyproject.toml
-SETUP_REQUIRES = ['setuptools >= 30.4.0']
-# This enables setuptools to install wheel on-the-fly
-SETUP_REQUIRES += ['wheel'] if 'bdist_wheel' in sys.argv else []
-
 if __name__ == '__main__':
-    setup(name='niworkflows',
-          version=versioneer.get_version(),
-          cmdclass=versioneer.get_cmdclass(),
-          setup_requires=SETUP_REQUIRES,
-          )
+    setupargs = {
+        "version": versioneer.get_version(),
+        "cmdclass": versioneer.get_cmdclass(),
+    }
+    if "bdist_wheel" in sys.argv:
+        setupargs["setup_requires"] = ["setuptools >= 38.3.0", "wheel"]
+    setup(**setupargs)
```

Comment:
I may have missed the discussion, but what warranted these changes in `setup.py`/`setup.cfg`/`pyproject.toml`? You mention bugs requiring 38.3.0, but they're not obvious to me just from looking at the code.

More generally, what motivated the refactor? This doesn't seem simpler to me, has removed the comments that can help future maintainers update it, and now that we've moved `name` into `setup.cfg`, GitHub's dependency graph won't work correctly with this repository.

Reply:
> now that we've moved `name` into `setup.cfg`, GitHub's dependency graph won't work correctly with this repository.

Didn't know that.

Anyways, I'll leave any revisions of the installation out (other than the better dealing with package_data).

Reply:
Yup, we should avoid `include_package_data = True`. It seemed like it made sense when I did that, but it means almost the exact opposite of how I interpreted it.